examples/graphics-2d/androidApp/build.gradle.kts

```diff
@@ -17,9 +17,9 @@ kotlin {
 
 android {
     compileSdk = 34
-    namespace = "org.jetbrains.fallingballs"
+    namespace = "org.jetbrains.Graphics2D"
```

Comment:
wrong case, should be org.jetbrains.graphics2d -- else Android version does not start